stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/avatar/UserAvatar.kt

```diff
@@ -110,23 +110,23 @@ public enum class OnlineIndicatorAlignment(public val alignment: Alignment) {
 @Preview
 @Composable
 private fun UserWithImageAvatarPreview() {
-    buildUserAvatarPreview(PreviewUserData.userWithImage)
```

Comment:
I think we should extract `OnlineIndicator(modifier: Modifier = Modifier)` **(line 88)** into a separate file as it's used both in UserAvatar and ChannelAvatar and is a publicly exposed function. 

This would run more inline with the new organization.

Reply:
Yeah definitely - I didn't dive into the components already in the components package, should've ran through them